tests/infra_test.go

```diff
@@ -666,120 +669,142 @@ var _ = Describe("Infrastructure", func() {
 				Transport: tr,
 			}
 
-			errors := make(chan error)
-			for ix := 0; ix < concurrency; ix++ {
-				go func(ix int) {
-					req, _ := http.NewRequest("GET", metricsURL, nil)
-					resp, err := client.Do(req)
-					if err != nil {
-						fmt.Fprintf(GinkgoWriter, "client: request: %v #%d: %v\n", req, ix, err) // troubleshooting helper
-					} else {
-						resp.Body.Close()
-					}
-					errors <- err
-				}(ix)
-			}
+			for _, url := range metricsURLs {
+				if urlIsIPv6(url) {
+					By("Skip Scraping the Prometheus endpoint (IPv6) until issue https://github.com/kubevirt/kubevirt/issues/4145 is fixed")
+					continue
+				}
+				By(fmt.Sprintf("Scraping the Prometheus endpoint %s", url))
+
```

Comment:
please remove the empty line, as `By(fmt.Sprintf("Scraping the Prometheus endpoint %s", url))` is more related to the below code than to the above one